bench/wtperf/wtperf.c

```diff
@@ -169,8 +169,9 @@ update_value_delta(WTPERF_THREAD *thread, int64_t delta)
         value[len + delta] = '\0';
     else {
         /* Extend the value by the configured amount. */
-        for (new_len = len; new_len < opts->value_sz_max && new_len - len < delta; new_len++)
+        for (new_len = len; new_len <= len + delta; ++new_len)
             value[new_len] = 'a';
+        value[new_len] = '\0';
```

Comment:
I feel we have a bug here. We need to write the nul terminator. @ddanderson

Reply:
@quchenhao Nice find! That fix looks correct to me.